datasets/doc/source/how-to-use-partitioners.ipynb

```diff
@@ -0,0 +1,257 @@
+       "Dataset({\n",
+       "    features: ['img', 'label'],\n",
+       "    num_rows: 5000\n",
+       "})"
+      ]
+     },
+     "execution_count": null,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from flwr_datasets import FederatedDataset\n",
+    "\n",
+    "fds = FederatedDataset(dataset=\"cifar10\", partitioners={\"train\": partitioner})\n",
+    "iid_partition = fds.load_partition(partition_id=0)\n",
+    "iid_partition"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'img': [<PIL.PngImagePlugin.PngImageFile image mode=RGB size=32x32>,\n",
+       "  <PIL.PngImagePlugin.PngImageFile image mode=RGB size=32x32>,\n",
+       "  <PIL.PngImagePlugin.PngImageFile image mode=RGB size=32x32>],\n",
+       " 'label': [1, 2, 6]}"
+      ]
+     },
+     "execution_count": null,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Let's take a look at the first three samples\n",
+    "iid_partition[:3]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using Different `Partitioners`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Why would you need to use different `Partitioner`s?**\n",
+    "\n",
+    "There are a few way that the data partitioning is simulated in the literature, `Flower Datasets` let's you work with the different approaches that have been proposed so far. If enable you to simulate partitions with different properties and different levels of heterogeneity.\n",
+    "\n",
+    "\n",
+    "**How to use different `Partitioner`s?**\n",
+    "\n",
+    "To use a different `Partitioner` you just need to create a different object (note it has typically different parameters that you need to specify). Then you pass as before to the `FederatedDataset`.\n",
+    "\n",
+    "<div style=\"max-width:80%; margin-left: auto; margin-right: auto\">\n",
+    "    <img src=\"./_static/tutorial-quickstart/partitioner-flexibility.png\" alt=\"Partitioner flexibility display\">\n",
+    "</div>\n",
+    "See the only changing part in yellow.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `PathologicalPartitioner`\n",
+    "\n",
+    "Now, we are going to create partitions that have only a subset of labels per in each partition."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Dataset({\n",
+       "    features: ['img', 'label'],\n",
+       "    num_rows: 2501\n",
+       "})"
+      ]
+     },
+     "execution_count": null,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from flwr_datasets.partitioner import PathologicalPartitioner\n",
```

Comment:
This section needs a bit of extra comments. In general, wouldn't it be better to rename this section to something like: "Creating non-IID partitions" ? A low alpha would also create a setting valid that meet the description given in the first markdown cell

Reply:
I renamed it into two parts (so it first includes your suggestion). I've also added a new section on Dirichlet